VS/VehicleSignals.rdf

```diff
@@ -1033,6 +1036,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;CrankshaftPositionSensor">
 		<rdfs:subClassOf rdf:resource="&auto-vs;VehicleSensor"/>
+		<rdfs:label xml:lang="en">Crankshaft Position Sensor</rdfs:label>
```

Comment:
Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.